pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -552,20 +552,24 @@ func testCNIUpdateKubeletArgs(t *testing.T) {
 	})
 }
 
-// TestPodManifestDirCreation tests if the pod manifest directory was created
-func TestPodManifestDirCreation(t *testing.T) {
+// TestKubeletDirectoriesCreation tests if the directories needed for Kubelet are initialized as required
+func TestKubeletDirectoriesCreation(t *testing.T) {
```

Comment:
I think, we decided that the same log directory would be used by hybrid overlay in a subsequent PR, so should we limit the name to `TestKubeletDirectoriesCreation` or something like to reflect all the directories created by WMCB?

Reply:
This function tests `initializeKubeletFiles` method (https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/bootstrapper/bootstrapper.go#L339). I believe we should keep it limited to the function it is testing for now. It can always be repurposed, if needed, in subsequent PRs

Reply:
No, what I meant was change the name to `TestInitializeKubeletFiles` then, when you rename the function to `initializeKubeAndCNIDir` then it makes sense to change the name of test to something appropriate
.

Reply:
I am fine with this change but I don't understand the comment about renaming the function to `initializeKubeAndCNIDir`. @ravisantoshgudimetla which function is being renamed?

Reply:
Initially we thought, log dir which will be generated by wmcb is also being used by hybrid-overlay not just kubelet, so it doesn't make sense to call it as just `initializeKubeletDir`, hence my previous comment, now we limited the scope to create a log directory just for kubelet, so we can ignore my comment